config/config.go

```diff
@@ -75,12 +76,13 @@ func get() (*Config, error) {
 		DefaultMaximumSearchResults:             500,
 		DefaultOffset:                           0,
 		DefaultPage:                             1,
+		DefaultAggregationSort:                  "release_date",
 		DefaultSort:                             "relevance",
 		DefaultDatasetSort:                      "release_date",
 		EnableCensusTopicFilterOption:           false,
 		EnableCensusPopulationTypesFilterOption: false,
 		EnableCensusDimensionsFilterOption:      false,
-		EnableReworkedDataAggregationPages:      false,
+		EnableReworkedDataAggregationPages:      true,
```

Comment:
This branch probably needs a rebase :) 
```suggestion
		EnableReworkedDataAggregationPages:      false,
```